agent/agentlocal/deps.go

```diff
@@ -23,8 +23,8 @@ import (
 	"github.com/percona/pmm/api/agentpb"
 )
 
-//go:generate ../../bin/mockery -name=client -case=snake -inpkg -testonly
-//go:generate ../../bin/mockery -name=supervisor -case=snake -inpkg -testonly
+//go:generate mockery -name=client -case=snake -inpkg -testonly
+//go:generate mockery -name=supervisor -case=snake -inpkg -testonly
```

Comment:
Instead of `mockery` in our tools folder here we will use mockery from that `PATH`. It may be a completely different version. I propose to keep exact path to tools binaries.

Reply:
We have conflicts on macs because of that. If we run `make init` locally and inside dev container they generate tools for different architectures.

Reply:
I remember we fixed this problem

Ah, ok, that was broken again in that PR: https://github.com/percona/pmm/pull/1003/files

Line 57 of `docker-compose.yml`

Reply:
So let's just revert this change, this fake volume will let us to have different set of tools inside container and on the host machine

Reply:
having `/root/go/src/github.com/percona/pmm/bin` in volumes doesn't fix it for local runs, it just rewrites the directory.

Reply:
Mmm, what do you mean? Local run will use local `bin`, container run will use container's `bin`

Reply:
it works if you have long running containers, but if you re-create it each time. it rewrites local directory on setup.

Reply:
So problem is that you start new containers frequently, and you don't want to install tools each time?

Reply:
Yes